cgmanifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "registrations": [
+    {
+      "component": {
+        "type": "git",
+          "git": {
+            "repositoryUrl": "https://github.com/atteneder/glTFast",
+            "commitHash": "04962b47618cdab46943217d30c6c6de33d3c3f7"
```

Comment:
This doesn't appear to match the hash in the package lock file. Adding the version explicitly to the package manifest should help fix this.

Reply:
the manifest.json has been updated and this commit hash should match now